runtime/base/php-74.Dockerfile

```diff
@@ -128,7 +129,7 @@ RUN /tmp/clean.sh && rm /tmp/clean.sh
 # Now we start back from a clean image.
 # We get rid of everything that is unnecessary (build tools, source code, and anything else
 # that might have created intermediate layers for docker) by copying online the /opt directory.
-FROM amazonlinux:2018.03
+FROM amazonlinux:2
```

Comment:
Here we switch from `amazonlinux:2018.03` to `amazonlinux:2` and for PHP 7.3 and 8.0 we switch to `lambci/lambda:build-provided.al2`.

@shouze do you remember the reason? I'm not sure which one to keep in the end (to make this consistent).

Reply:
Good catch! We should keep `amazonlinux:2` everywhere for this build stage. lambci one is a heavy docker image:
![image](https://user-images.githubusercontent.com/54712/100588783-8e8cd200-32f2-11eb-9df5-eabac93502dd.png)
amazonlinux one is lighter, and more conform to the true production amazon environment:
![image](https://user-images.githubusercontent.com/54712/100588955-bd0aad00-32f2-11eb-9a81-2d4ec4999309.png)

Reply:
Just in time before new year's eve! :clock1130:  https://github.com/brefphp/bref/pull/830